[extractor/niconicochannelplus] Add extractor (#5686)
yt-dlp/yt-dlp: README.md

* `vcodec`: vcodec to ignore - one or more of `h264`, `h265`, `dvh265`
* `dr`: dynamic range to ignore - one or more of `sdr`, `hdr10`, `dv`

#### niconicochannelplus

> * `max_comments`: Limit the amount of comments to gather (default: `120`)

Reviewer: ```suggestion
* `max_comments`: Maximum number of comments to extract - default is `120`
```
